workflows4s-cats-effect/src/main/scala/workflows4s/runtime/instanceengine/WorkflowInstanceEngineBuilder.scala

```diff
@@ -0,0 +1,77 @@
+package workflows4s.runtime.instanceengine
+
+import cats.effect.IO
+import workflows4s.catseffect.CatsEffect.given
+import workflows4s.effect.Effect
+import workflows4s.runtime.registry.WorkflowRegistry
+import workflows4s.runtime.wakeup.KnockerUpper
+
+import java.time.Clock
+
+object WorkflowInstanceEngineBuilder {
```

Comment:
This probably belongs to core, with `withCatsTime` as an extension. Or we can think of alternative API, but the point stands